### DISCUSSION TITLE
Keystore file not set for signing config release

### DISCUSSION THREAD

**Discussion by danielgsaavedra** [pavjacko/renative#948] [General]
Hello community, I have a problem trying to build a release in my android tv application, I have tried configuring the renative.private.json, also the renative.json of the appConfigs folder as follows:
```
{
   "signingConfigs" : {
     "release": {
       "storeFile": "~/release.keystore.jks",
       "storePassword": "pass",
       "keyAlias": "app-androidtv",
       "keyPassword": "pass"
     }
   }
} 
```
and also like this:
```
{ 
  "androidtv": {
    "storeFile": "~/release.keystore.jks",
    "storePassword": "pass",
    "keyAlias": "app-androidtv",
    "keyPassword": "pass"
  }
}
```
But I have not been successful, and in the documentation I cannot find an exact guide to do this process, if someone could help me it would be very helpful. Beforehand thank you very much.

**Comment by pauliusguzas** [comment]
hello @danielgsaavedra , renative.private.json should look something like this. If you still have any questions feel free to ask
```
{
    "platforms": {
        "androidtv": {
            "storeFile": "./release.keystore",
            "storePassword": "pass",
            "keyAlias": "key",
            "keyPassword": "pass"
        }
    }
}
```

**Comment by pauliusguzas** [comment]
@danielgsaavedra closing this discussion as https://github.com/flexn-io/renative/issues/949 ticket is already there with answer
